docker/Dockerfile.cpu

```diff
@@ -16,17 +16,17 @@ RUN apt-get update && \
     && curl -O https://bootstrap.pypa.io/get-pip.py \
     && python get-pip.py
 
-RUN pip install mxnet-model-server \
-    && pip install mxnet-mkl
+RUN pip install --no-cache-dir mxnet-model-server \
+    && pip install --no-cache-dir mxnet-mkl==1.3.0
```

Comment:
Should we hardcode this version ? 
Would pip install mxnet-mkl not fetch whichever the latest mxnet version is ?

Reply:
Here's another reference that we can have a look at : 
How mxnet version is being handled for generating Docker images. 
https://github.com/apache/incubator-mxnet/pull/12791/files#diff-36826cee27ee722a559d139fc7230c28R22

Reply:
This is a release Docker file, I keep it as latest in our nightly builds. I think we should hardcode the version of MXNet so that the container environment for our release-build is known.

Reply:
@piyushghai : That link is interesting :) . Thanks for sharing it..